src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -115,18 +118,96 @@ export const DownloadCartPage:React.FunctionComponent<DownloadCartPageProps> = (
       {selectedTabIndex == 1 && !isError && !isLoading && data &&
         <div>
           {data.numberOfFilesAvailableForDownload > 0 && 
-            <div>
+            <div className="DownloadListTabContent">
               <div className="subSectionOverview">
-                <div className="container">
-                  <div className="subSectionContainer">
-                    <span className="subSectionTitle">Complete Your Download</span>
-                    <DownloadListStats numBytes={data.sumOfFileSizesAvailableForDownload} numPackagableFiles={data.numberOfFilesAvailableForDownloadAndEligibleForPackaging} numFiles={data.numberOfFilesAvailableForDownload}/>
+                <div>
+                  <div className="headlineWithHelp">
+                    <Typography variant={'headline3'}>
+                      <IconSvg options={{ icon: 'packagableFile' }} /> Web Download (.Zip Packages)
+                    </Typography>
+                    <MarkdownPopover
+                      contentProps={{ markdown: 'This will allow you to create a .zip file that contains eligible files. Files greater that 100 MB, external links, or files which are not stored on Synapse native storage are ineligible. In most cases, ineligible files can be downloaded individually. External links will require navigation to an external site, which may require a seperate login process.' }}
```

Comment:
I was thinking we should link to docs here but relevant docs don't exist, so I created a ticket: https://sagebionetworks.jira.com/browse/DOCS-98